flags.go

```diff
@@ -381,7 +381,7 @@ func newApp() (app *cli.App) {
 			cli.StringFlag{
 				Name:  MetadataPrefetchOnMountFlag,
 				Value: config.DefaultMetadataPrefetchOnMount,
-				Usage: "This indicates whether or not to prefetch the metadata (prefilling of metadata caches and creation of inodes) of the mounted bucket at the time of mounting the bucket. Supported values: \"disabled\", \"sync\" and \"async\". Any other values will return error on mounting. This is applicable only to single-bucket mount-points, and not to dynamic-mount points.",
+				Usage: "Experimental: This indicates whether or not to prefetch the metadata (prefilling of metadata caches and creation of inodes) of the mounted bucket at the time of mounting the bucket. Supported values: \"disabled\", \"sync\" and \"async\". Any other values will return error on mounting. This is applicable only to single-bucket mount-points, and not to dynamic mount-points.",
```

Comment:
nit: single-bucket mount-points, and not to dynamic mount-points. -> static mounting and not on dynamic mounting.

Reply:
done